docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+| `fuzzymatching=` | `true` \| `false`             | 0..1          | If true fuzzy matching is applied to PatientName attribute. It will do a prefix word match of any name part inside PatientName value. For example, if PatientName is "John^Doe", then "joh", "do", "jo do", "Doe" and "John Doe" will all match. However "ohn" will not match. |
+
+#### Searchable Attributes
+
+We support searching on below attributes and search type.
+
+| Attribute Keyword | All Studies | All Series | All Instances | Study's Series | Study's Instances | Study Series' Instances |
+| :---------------- | :---: | :----: | :------: | :---: | :----: | :------: |
+| StudyInstanceUID | X | X | X |  |  |  |
+| PatientName | X | X | X |  |  |  |
+| PatientID | X | X | X |  |  |  |
+| PatientBirthDate | X | X | X |  |  |  |
+| AccessionNumber | X | X | X |  |  |  |
+| ReferringPhysicianName | X | X | X |  |  |  |
+| StudyDate | X | X | X |  |  |  |
+| StudyDescription | X | X | X |  |  |  |
+| ModalitiesInStudy | X | X | X |  |  |  |
+| SeriesInstanceUID |  | X | X | X | X |  |
+| Modality |  | X | X | X | X |  |
+| PerformedProcedureStepStartDate |  | X | X | X | X |  |
+| ManufacturerModelName | | X | X | X | X |  |
+| SOPInstanceUID |  |  | X |  | X | X |
+
+#### Search Matching
+
+We support below matching types.
+
+| Search Type | Supported Attribute | Example |
+| :---------- | :------------------ | :------ |
+| Range Query | StudyDate, PatientBirthDate | {attributeID}={value1}-{value2}. For date/ time values, we support an inclusive range on the tag. This will be mapped to `attributeID >= {value1} AND attributeID <= {value2}`. If {value1} is not specified, all occurrences of dates/times prior to and including {value2} will be matched. Likewise, if {value2} is not specified, all occurrences of {value1} and subsequent dates/times will be matched. However, one of these values has to be present. `{attributeID}={value1}-` and `{attributeID}=-{value2}` are valid, however, `{attributeID}=-` is invalid. |
+| Exact Match | All supported attributes | {attributeID}={value1} |
+| Fuzzy Match | PatientName, ReferringPhysicianName | Matches any component of the name which starts with the value. |
+
+#### Attribute ID
+
+Tags can be encoded in a number of ways for the query parameter. We have partially implemented the standard as defined in [PS3.18 6.7.1.1.1](http://dicom.nema.org/medical/dicom/2019a/output/chtml/part18/sect_6.7.html#sect_6.7.1.1.1). The following encodings for a tag are supported:
+
+| Value            | Example          |
+| :--------------- | :--------------- |
+| {group}{element} | 0020000D         |
+| {dicomKeyword}   | StudyInstanceUID |
+
+Example query searching for instances:
+`../instances?Modality=CT&00280011=512&includefield=00280010&limit=5&offset=0`
+
+### Search Response
+
+The response will be an array of DICOM datasets. Depending on the resource, by *default* the following attributes are returned:
+
+#### Default Study tags
```

Comment:
Were there intentional changes here?  This table is much smaller than the [published V1](https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/dicom-services-conformance-statement#default-study-tags) - I noted these are missing:

| Tag          | Attribute Name |
| :----------- | :------------- |
| (0008, 0005) | `SpecificCharacterSet` |
| (0008, 0030) | `StudyTime` |
| (0008, 0056) | `InstanceAvailability` |
| (0008, 0201) | `TimezoneOffsetFromUTC` |
| (0010, 0040) | `PatientSex` |
| (0020, 0010) | `StudyID` |

And this appears to be new in V2 (or missing in V1)

| Tag          | Attribute Name |
| :----------- | :------------- |
| (0008, 1030) | StudyDescription |

Reply:
Yes, reduce the default. Sorry, there was a Code change PR with more details and since this was a follow up did not put a lot in description. Will keep this in mind next time